proto/pki/tx.proto

```diff
@@ -21,51 +21,51 @@ service Msg {
 
 message MsgProposeAddX509RootCert {
   string signer = 1 [(cosmos_proto.scalar) = "cosmos.AddressString", (gogoproto.moretags) = "validate:\"required\""];
-  string cert = 2 [(gogoproto.moretags) = "validate:\"required\""];
+  string cert = 2 [(gogoproto.moretags) = "validate:\"required,max=3500000\""];
 }
 
 message MsgProposeAddX509RootCertResponse {
 }
 
 message MsgApproveAddX509RootCert {
   string signer = 1 [(cosmos_proto.scalar) = "cosmos.AddressString", (gogoproto.moretags) = "validate:\"required\""];
-  string subject = 2 [(gogoproto.moretags) = "validate:\"required\""];
-  string subject_key_id = 3 [(gogoproto.moretags) = "validate:\"required\""];
+  string subject = 2 [(gogoproto.moretags) = "validate:\"required,max=50\""];
```

Comment:
Too small. Look at https://crypto.stackexchange.com/questions/54148/does-maximum-length-of-cn-in-a-x509-cert-include-the-length-of-the-oids and https://www.rfc-editor.org/rfc/rfc3280#section-4.1.2.6.
Let's restrict to some reasonably high value. For example, `1024` (1KB)